Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -996,6 +996,11 @@ struct WebCore::ApplePayDeferredPaymentRequest {
 };
 #endif
 
+#if ENABLE(APPLE_PAY_DISBURSEMENTS)
+struct WebCore::ApplePayDisbursementPaymentRequest {
```

Comment:
Not sure if this should be empty.

Reply:
Same as above, these will have properties eventually (for now, simply having a `disbursementPaymentRequest` property existing is sufficient to represent a disbursement.